src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--31 calendar--start-day-mon">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--31 calendar--start-day--mon">
```
define modifier for the start day in this way